scraper/settings.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
```

Comment:
this is the default settings.py file with some minor modifications by me to enable the DeDupePipeline and the SubdomainBlockerMiddleware